apps/web/src/pages/templates/components/SmsBubble.tsx

```diff
@@ -0,0 +1,159 @@
+import styled from '@emotion/styled';
+import { Skeleton } from '@mantine/core';
+import { colors, PencilOutlined } from '@novu/design-system';
+import { MouseEventHandler, useState } from 'react';
+
+const SmsBubbleHolder = styled.div`
+  position: relative;
+`;
+
+const SmsBubbleContainer = styled.div<{ isBlur: boolean; isError: boolean }>`
+  position: relative;
+  width: fit-content;
+  min-width: 160px;
+  min-height: 36px;
```

Comment:
set the min-width because on the hover we show blurred background and edit button which needs some space